types/stylelint/index.d.ts

```diff
@@ -279,147 +279,159 @@ declare namespace stylelint {
 	};
 
 	type StringOrRegex = string | RegExp;
-	type StringsOrRegexes = Array<StringOrRegex>;
-	type OneOrMoreStrings = string | string[];
-	type Primary = number | true | StringOrRegex | StringsOrRegexes | Record<string, any>;
+	type OneOrMany<S> = S | S[];
```

Comment:
`type-fest` calls it [`Arrayable`](https://github.com/sindresorhus/type-fest/blob/main/source/arrayable.d.ts).